packages/salesforcedx-vscode-core/README.md

```diff
@@ -71,6 +71,18 @@ After you run Apex tests, two new commands are available in the command palette:
 
 To retrieve code coverage results when you run Apex tests, edit your workspace settings and set `salesforcedx-vscode-core.retrieve-test-code-coverage` to `true`.  
 
+## Explore Your Apex Tests
+
+The Apex Tests sidebar provides several features. Here, you can see all your Apex tests at a glance. You can run one test method, the test methods in one class, or all your tests. You can view the results of your last test run. And, you can jump from those results to the corresponding lines in your code. To access this sidebar, click the flask icon (hover text: Test) in the view bar on the left side of the VS Code window. (If you don’t see this icon, make sure that the project you have open in VS Code contains an `sfdx-project.json` file in its root directory.)  
+
+To run selected tests, in the Apex Tests view, hover over the name of a test method or class to reveal a play icon. Click the play icon (hover text: Run Single Test) to run a test method or all the methods in a class. To run all your tests, click the larger play icon at the top of the Apex Tests view (hover text: Run Tests).  
+
+After you run tests, the blue icons next to your classes and methods change to green icons (for passing tests) or red icons (for failing tests). To see details about your test runs, hover over the name of a test class in the sidebar.
+
+To jump to the definition of a test class, a test method that passed, or a method that you haven’t run yet, click its name in the sidebar. If you click the name of a failed test method, you jump to the assert statement that failed.  
```

Comment:
"...a test method that passed, or a method that you haven’t run yet..." <-- Might be easier to just say definition of a test method since we allow you to jump to it even if you haven't run it; it ran and passed; or if it ran and failed or errored out.